docs/guides/execution-modes.mdx

```diff
@@ -5,73 +5,169 @@ description: An overview of the available execution modes in Qiskit Runtime; ses
+
+To run in batch mode, specify `mode=batch` when instantiating a primitive or run the job in a batch context manager. See [Run jobs in a batch](/guides/run-jobs-batch) for examples.
+
+## Session mode
+
+A dedicated window for running a multi-job workload. This allows users to experiment with variational algorithms in a more predictable way and even run multiple experiments simultaneously, taking advantage of parallelism in the stack. Using sessions helps avoid delays caused by queuing each job separately, which can be particularly useful for iterative tasks that require frequent communication between classical and quantum resources.
+
+ To run in session mode, specify `mode=session` when instantiating a primitive, or run the job in a session context manager. See [Run jobs in a session](/guides/run-jobs-session) for examples.
+
+ <Admonition type="note">
+Session execution mode is not supported in the Open Plan. Jobs will run in job mode instead.
 </Admonition>
 
-**Job mode**: A single primitive request of the estimator or the sampler made without a context manager. Circuits and inputs are packaged as primitive unified blocs (PUBs) and submitted as an execution task on the quantum computer. To run in job mode, specify `mode=backend` when instantiating a primitive. See [Primitives examples](primitives-examples) for examples.
+## How long do workloads run?
+
+The amount of time an execution runs depends on the values for maximum execution time (or _time to live_ (TTL)), interactive timeout (or _interactive time to live_ (ITTL)), and, for Qiskit Runtime on IBM Cloud&reg;, the specified Cost limit.
+
+**Maximum execution time (TTL)** - Determines how long (in _quantum time_) a session or batch can run.  This is the smaller of the value set for `max_execution_time` and the QPU-determined job timeout value.  Quantum time is the time spent by the QPU complex to process the workload. This timer starts when the first job in the workload starts being processed and continues running until the value is reached. After this value is reached, the workload is terminated, any jobs that are already running continue running, and any queued jobs that remain in the session or batch are put into a failed state. You can set this value with the `max_time` parameter for [batches](/guides/run-jobs-batch#specify-batch-length) or [sessions](/guides/run-jobs-session#specify-length).
+
+**Interactive timeout (ITTL)** - This value cannot be configured. If no jobs are queued for the active session or batch within that window, the workload is temporarily deactivated. A job submitted to the session or batch reactivates the deactivated workload if the maximum timeout value has not been reached.
+
+For full details about these values, including how to determine the ITTL value, review the [Maximum execution time guide.](/guides/max-execution-time)
```

Comment:
I find it a bit confusing that when reading this guide on execution mode, there is a link to the max execution time page, which has a link back to this guide...

Reply:
> which has a link back to this guide

I didn't find a link to this guide from the Max execution time one.

Reply:
It's the [Specify the batch length.](https://docs.quantum.ibm.com/guides/run-jobs-batch#specify-batch-length) (and session length). They don't link back to this specific page but to the execution mode section.

Reply:
restructured some stuff.